Kernel/Arch/aarch64/TrapFrame.h

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2018-2021, Andreas Kling <kling@serenityos.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/Types.h>
+
+#include <AK/Platform.h>
+
+namespace Kernel {
+
+struct TrapFrame {
```

Comment:
This commit should remove it from Kernel/Arch/aarch64/init.cpp too and add the include, so that this commit here is a self-contained and complete "Move TrapFrame to its own header" change.

Reply:
Done!